examples/cookbook/design/themes/lib/main.dart

```diff
@@ -11,30 +14,39 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     const appName = 'Custom Themes';
 
-    // #docregion MaterialApp
     return MaterialApp(
       title: appName,
       theme: ThemeData(
-        // Define the default brightness and colors.
-        brightness: Brightness.dark,
-        primaryColor: Colors.lightBlue[800],
+        useMaterial3: true,
 
-        // Define the default font family.
-        fontFamily: 'Georgia',
+        // Define the default brightness and colors.
+        colorScheme: ColorScheme.fromSwatch(
```

Comment:
We don't recommend using `fromSwatch` (M1/M2 concept) but rather `fromSeed` (M3+)

Reply:
Making these changes return very unusual results. I would like to address this separately.

Reply:
Sure, can you file an issue so this doesn't get lost (or modify @Piinks' issue?